Kickstarter-iOS/AppDelegate.swift

```diff
@@ -249,6 +249,16 @@ internal final class AppDelegate: UIResponder, UIApplicationDelegate {
 
     self.window?.tintColor = .ksr_green_700
 
+    self.viewModel.outputs.goToOnboarding
+      .observeForControllerAction()
+      .observeValues { [weak self] in
+
+        let navController = NavigationController(rootViewController: CategorySelectionViewController())
```

Comment:
Think for most of our programmatic VCs we've conformed them to `Instantiable` and used `.instantiate()` right? That calls the designated initializer.